src/map/handler/TouchGestures.js

```diff
@@ -87,12 +96,15 @@ L.Map.TouchGestures = L.Handler.extend({
             p2 = map.mouseEventToContainerPoint(e.touches[1]),
             vector = p1.subtract(p2),
             scale = p1.distanceTo(p2) / this._startDist,
+            inertia = map.options.touchRotateIntertia
             delta;
 
         if (this._rotating) {
             var theta = Math.atan(vector.x / vector.y);
             var bearingDelta = (theta - this._startTheta) * L.DomUtil.RAD_TO_DEG;
             if (vector.y < 0) { bearingDelta += 180; }
+            if (inertia && bearingDelta > inertia) { bearingDelta = 0; }
```

Comment:
```suggestion
            if (inertia && bearingDelta <= inertia) { bearingDelta = 0; }
```

(Just by looking at the code) Shouldn't it be `<` instead of `>`?

Reply:
@mauriciabad I don't own any touch device, so please tell me which one works properly.

Reply:
It was indeed flipped